contrib/spice-gtk/template.py

```diff
@@ -0,0 +1,67 @@
+pkgname = "spice-gtk"
+pkgver = "0.42"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dcoroutine=libucontext",
+    "-Dwayland-protocols=enabled",
+    "-Dlibcap-ng=enabled",
+    "-Dgtk=enabled",
+    "-Degl=enabled",
+    "-Dlz4=enabled",
+    "-Dopus=enabled",
+    "-Dsasl=enabled",
+    "-Dvapi=enabled",
+    "-Dsmartcard=enabled",
+    "-Dintrospection=enabled",
+    "-Dbuiltin-mjpeg=false",
+    "-Dpolkit=disabled",  # uses getuid binaries
```

Comment:
what?

Reply:
iirc the polkit-helpers binaries are installed as setuid root for some reason

Reply:
oh i see what you mean  
i wrote `getuid` instead of `setuid`

Reply:
typo fixed.

i assume that's all that needed to be fixed here?